tools/swift-api-digester/ModuleAnalyzerNodes.cpp

```diff
@@ -1072,7 +1076,10 @@ SDKNodeInitInfo::SDKNodeInitInfo(SDKContext &Ctx, ValueDecl *VD)
   if (auto *CD = dyn_cast_or_null<ClassDecl>(VD)) {
     if (auto *Super = CD->getSuperclassDecl()) {
```

Comment:
It would be better to use the superclass type (getSuperclass()) and not the superclass declaration (getSuperclassDecl()) in case you change something like `class Derived : Base<Int>` to `class Derived : Base<String>`.

Reply:
This commit should address generic argument change in super classes: https://github.com/apple/swift/pull/19539/commits/cf03b247d622199ad4adfd7a0879dc40b15b1686